tests/org.corpus_tools.hexatomic.core.tests/src/main/java/org/corpus_tools/hexatomic/core/DummySync.java

```diff
@@ -12,4 +12,19 @@ public void syncExec(Runnable runnable) {
   public void asyncExec(Runnable runnable) {
     runnable.run();
   }
+
+  @Override
+  protected boolean isUIThread(Thread thread) {
+    return false;
+  }
+
+  @Override
+  protected void showBusyWhile(Runnable runnable) {
+    // Not implemented in dummy
+  }
+
+  @Override
+  protected boolean dispatchEvents() {
+    return false;
+  }
 }
```

Comment:
Add newline so as to not trip git.

```suggestion
}

```